lib/discovery.js

```diff
@@ -196,7 +197,7 @@ function mixinDiscovery(DB2, db2) {
     //   options = {};
     // }
     if (typeof options !== 'object') {
-      throw new Error('options must be an object: ' + options);
+      throw new Error(g.f('options must be an {{object}}: %s'), options);
```

Comment:
Should be `throw new Error(g.f('options must be an {{object}}: %s', options));`
End bracket in the wrong place.